src/Disks/VolumeJBOD.cpp

```diff
@@ -76,6 +76,7 @@ VolumeJBOD::VolumeJBOD(
     perform_ttl_move_on_insert = config.getBool(config_prefix + ".perform_ttl_move_on_insert", true);
 
     are_merges_avoided = config.getBool(config_prefix + ".prefer_not_to_merge", false);
+    least_used_ttl_ms = config.getUInt64(config_prefix + ".least_used_ttl_ms", 60);
```

Comment:
If it's `_ms` then should the default value be 60000 if you meant 60 seconds?

Reply:
Yep, nice catch, thanks.